treeherder/webapp/api/performance_data.py

```diff
@@ -64,7 +64,14 @@ def list(self, request, project):
                 framework__in=frameworks)
 
         interval = request.query_params.get('interval')
```

Comment:
Thanks for the feedback! Alright I am a bit confused. So currently interval is used basically to calculate the start date, i.e. get data for the current date/time minus the interval i.e. for the last X days.

If we use a start_date and and an end_date, I'm unclear as to why the interval would be required, since you already have the start and end dates. Sorry if I am misunderstanding!